targets/csharp/source/PlayFabSDK/UnittestRunner/UnittestRunner.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>netcoreapp1.1;netcoreapp2.0;net45</TargetFrameworks>
```

Comment:
Are the netcoreapp additions intentional?

Reply:
Yup, netcoreapp 1.1 and 2.0 apparently have hit end of life :( 

https://dotnet.microsoft.com/download/dotnet/2.0/runtime?utm_source=getdotnetcore&utm_medium=referral 

I got a hang when running 1.1, and 2.0 spits our this url that mentions this is no longer supported by microsoft.